ansible/roles/media-licenses/tasks/main.yml

```
@@ -0,0 +1,16 @@
+- name: media licenses sync CronJob ConfigMap
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: vidis-sync-cronjob-configmap.yml.j2
+  tags:
+    - configmap
+
+- name: media licenses sync CronJob
+  when: WITH_MEDIA_LICENSES is defined and WITH_MEDIA_LICENSES|bool == true
+  kubernetes.core.k8s:
+    kubeconfig: ~/.kube/config
+    namespace: "{{ NAMESPACE }}"
+    template: vidis-sync-cronjob.yml.j2
```

To add or Remove this cronjob on condition it's easyer to set the state 
```suggestion
  kubernetes.core.k8s:
    kubeconfig: ~/.kube/config
    namespace: "{{ NAMESPACE }}"
    template: vidis-sync-cronjob.yml.j2
    state: "{{ 'present' if WITH_MEDIA_LICENSES is defined and WITH_MEDIA_LICENSES|bool else 'absent'}}"
```